contrib/qubes/README.md

```diff
@@ -0,0 +1,243 @@
+# qubes-fwupd
+
+fwupd wrapper for QubesOS
+
+## Table of Contents
+
+* [Requirements](#Requirements)
+* [Usage](#Usage)
+* [Installation](#Installation)
+* [Testing](#Testing)
+* [Whonix support](doc/whonix.md)
+* [UEFI capsule update](doc/uefi_capsule_update.md)
+* [Heads update](doc/heads_update.md)
+
+## OS Requirements
+
+**Operating System:** Qubes OS R4.1 or Qubes OS R4.0
```

Comment:
I think you need to update this to 4.1 only since you mentioned elsewhere in this thread you're going to only support that now, and probably the other versions below too then.

Reply:
Fixed.